actions1/cla_check.js

```diff
@@ -0,0 +1,96 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview File to check if PR Author has signed the CLA
+ */
+
+const core = require('@actions/core');
+const { context } = require('@actions/github');
+const { execSync } = require('child_process');
+const { google } = require('googleapis');
+
+const SHEETS_TOKEN = process.env.SHEETS_TOKEN;
+const CREDENTIALS = JSON.parse(process.env.SHEETS_CRED);
+const SPREADSHEET_ID = '1naQC7iEfnro5iOjTFEn7iPCxNMPaPa4YnIddjT5CTM8';
+const RANGE = 'Usernames';
+const PR_AUTHOR = context.payload.pull_request.user.login;
+const PR_NUMBER = context.payload.pull_request.number;
+const LINK_RESULT = (
+  'https://github.com/oppia/oppia/wiki' +
+  '/Contributing-code-to-Oppia#setting-things-up');
+
+/**
+ * Create an OAuth2 client with the given credentials, and then execute the
+ * given claCheck function.
+ */
+const authorize = function(callback) {
+  // eslint-disable-next-line camelcase
+  const { client_secret, client_id, redirect_uris } = CREDENTIALS.installed;
+  const oAuth2Client = new google.auth.OAuth2(
+    client_id,
+    client_secret,
+    redirect_uris[0]
+  );
+  oAuth2Client.setCredentials(JSON.parse(SHEETS_TOKEN));
+  claCheck(oAuth2Client);
```

Comment:
Why you call this inside `authorise` I would expect it the other way, `authorise` being called inside `claCheck`.

Reply:
DONE